src/main/java/codeu/model/store/persistence/PersistentStorageAgent.java

```diff
@@ -103,4 +108,20 @@ public void writeThrough(Conversation conversation) {
   public void writeThrough(Message message) {
     persistentDataStore.writeThrough(message);
   }
+
+  public void deleteEntity(UUID id) {
+    //Key key = KeyFactory.generateKey(id.toString());
```

Comment:
Why are these two methods commented out?

Reply:
Whoops, I forgot I left those in. Originally I was figuring out how to delete the test data from the datastore, before I realized it would be better to just never write it in the first place.